kotsadm/web/src/components/PreflightResultPage.jsx

```diff
@@ -52,6 +51,20 @@ class PreflightResultPage extends Component {
     }
   }
 
+  deployKotsVersion = async (appSlug, sequence) => {
+    try {
+      await fetch(`${window.env.API_ENDPOINT}/app/${appSlug}/sequence/${sequence}/deploy`, {
+        headers: {
+          "Authorization": Utilities.getToken(),
+          "Content-Type": "application/json",
+        },
+        method: "POST",
+      });
+    } catch(err) {
+      console.log(err);
```

Comment:
wanna show errors in UI if request fails?

Reply:
since this pr just moves the logic from TS to GO, we can handle showing the error message in a separate pr.

Reply:
IRLed...updating later!